scripts/lib/update_acme_tests.py

```diff
@@ -355,11 +355,15 @@ def sort_by_time(test_one, test_two):
     """
     rec1, rec2 = get_recommended_test_time(test_one), get_recommended_test_time(test_two)
     if rec1 == rec2:
-        return cmp(test_one, test_two)
+        return (test_one > test_two) - (test_two < test_one)
     else:
         if rec2 is None:
             return -1
         elif rec1 is None:
             return 1
         else:
-            return cmp(convert_to_seconds(rec2), convert_to_seconds(rec1))
+            a = convert_to_seconds(rec2)
```

Comment:
This change confuses me.

Reply:
cmp was removed in python3, this function is equivalent.
https://stackoverflow.com/questions/15556813/python-why-cmp-is-useful

Reply:
Would it be worth having our own utils.cmp function that does this, since I agree with @jgfouca that this code is not intuitive?